recipe/meta.yaml

```diff
@@ -1,19 +1,18 @@
 {% set name = "dedalus" %}
-{% set version = "2.2006" %}
-
+{% set version = "2.2207" %}
 {% set mpi_prefix = "mpi_" + mpi %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7b6005c6c9b3b85c42da9a2ae85219b2722e2f61bda714d2518c4c058fd5d8df
+  url: https://files.pythonhosted.org/packages/6d/89/0f5b60cfd1d8dbe89f1f7293ee388ceef66f537ee8b5988cb6c7a8d35c6b/dedalus-2.2207.tar.gz
```

Comment:
Why was the url structure changed? With this change, the conda-forge bot won't detect version updates.

Reply:
It seems like it took a day for two for the short URL's to be updated by pypi, and this was failing when I first updated the recipe.  But it looks like that link is working now, so I changed it back in d598b7d.